package.json

```diff
@@ -28,53 +31,60 @@
-    "test": "eslint --max-warnings 0 --report-unused-disable-directives test/**/*.{js,jsx,mjs,ts,tsx}",
-    "validate:formatting": "prettier --check --no-editorconfig .",
-    "validate:linting:eslint": "eslint-config-prettier ./lib/eslintBaseConfig.json"
+    "list:todo-comments": "rg --only-matching '(TODO|FIXME)([\t ]+\\[[^\\]]+\\])?:[\\w\\s!\"#$%&'\\''()+,./:;<=>?@\\^_`{|}~-]+'",
+    "postinstall": "bun run --silent check:environment:versions",
+    "reinstall": "bun run --silent delete:package-lock && bun run --silent delete:node-modules && bun install",
+    "reinstall:use-lock-file": "bun run --silent delete:node-modules && bun install --frozen-lockfile",
+    "test": "eslint --max-warnings 0 --report-unused-disable-directives test/**/*.{js,jsx,mjs,ts,tsx}"
   },
   "dependencies": {
-    "@typescript-eslint/eslint-plugin": "6.21.0",
-    "@typescript-eslint/parser": "6.21.0",
+    "@typescript-eslint/eslint-plugin": "7.16.1",
+    "@typescript-eslint/parser": "7.16.1",
     "eslint-config-airbnb": "19.0.4",
     "eslint-config-airbnb-base": "15.0.0",
-    "eslint-config-airbnb-typescript": "17.1.0",
+    "eslint-config-airbnb-typescript": "18.0.0",
     "eslint-config-prettier": "9.1.0",
-    "eslint-plugin-compat": "4.2.0",
+    "eslint-plugin-compat": "6.0.0",
     "eslint-plugin-eslint-comments": "3.2.0",
     "eslint-plugin-import": "2.29.1",
-    "eslint-plugin-jest": "27.9.0",
-    "eslint-plugin-jest-dom": "5.1.0",
+    "eslint-plugin-jest": "28.6.0",
+    "eslint-plugin-jest-dom": "5.4.0",
     "eslint-plugin-jest-formatting": "3.1.0",
-    "eslint-plugin-jsx-a11y": "6.8.0",
-    "eslint-plugin-n": "16.6.2",
-    "eslint-plugin-playwright": "1.3.0",
-    "eslint-plugin-regexp": "2.2.0",
-    "eslint-plugin-security": "2.1.1",
-    "eslint-plugin-unicorn": "51.0.1"
+    "eslint-plugin-jsdoc": "48.7.0",
+    "eslint-plugin-jsx-a11y": "6.9.0",
+    "eslint-plugin-n": "17.9.0",
+    "eslint-plugin-playwright": "1.6.2",
+    "eslint-plugin-regexp": "2.6.0",
+    "eslint-plugin-security": "3.0.1",
+    "eslint-plugin-unicorn": "54.0.0"
   },
   "devDependencies": {
-    "@playwright/test": "1.41.2",
-    "@types/aws-lambda": "8.10.134",
-    "@types/node": "20.11.19",
-    "@types/react": "18.2.56",
-    "@types/react-dom": "18.2.19",
-    "eslint": "8.56.0",
+    "@mangs/bun-utils": "2.25.0",
+    "@playwright/test": "1.45.2",
+    "@types/aws-lambda": "8.10.141",
+    "@types/node": "20.14.11",
+    "@types/react": "18.3.3",
+    "@types/react-dom": "18.3.0",
+    "eslint": "8.57.0",
     "jest": "29.7.0",
-    "preact": "10.19.5",
-    "prettier": "3.2.5",
+    "preact": "10.22.1",
+    "prettier": "3.3.3",
     "prop-types": "15.8.1",
-    "react": "18.2.0",
-    "react-dom": "18.2.0",
-    "typescript": "5.3.3"
+    "react": "18.3.1",
+    "react-dom": "18.3.1",
+    "typescript": "5.5.3"
   },
   "peerDependencies": {
-    "eslint": ">=8.44.0"
+    "eslint": "^8.56.0"
```

Comment:
semver breaking change (version bump, and eslint 9 is not allowed due to airbnb incompatibility)